content/installation.md

```diff
@@ -150,9 +150,7 @@ gem install cucumber
 Add `cucumber` to your `Gemfile`:
 
 ```ruby
-group :test do
-  gem 'cucumber'
-end
+gem 'cucumber'
```

Comment:
This is the 10-minute installation and is using ruby as a standalone tool. Not as part of a server.

Reply:
Tutorials tend to set patterns that new users follow for a long time. Its better to have a bit more stuff for them to cut and paste rather than set a precedent that they probably will not have enough knowledge to unlearn for quite a while

Reply:
So the original `group :test do` gives it a test scope? (non-Ruby user ;) I'd prefer to keep that then.

Reply:
It depends. It has no use in regular ruby environments. There's a few things in here that aren't strictly wrong that I've fixed (Such as rockets e.t.c.)

For those there are 2 trains of thought. There isn't really a "right" answer. In my eyes I wouldn't ever teach people about using grouping inside bundler unless they were doing Rails. It has no use and just muddies the file.

Likewise, if you teach them to use grouping. Then when it comes to Rails they don't need to learn what it does.

My opinion is that because it doesn't actually do anything, they might expect it to, which is wrong.

My 0.02c